diff -ur aMule-2.1.0.org/configure.in aMule-2.1.0/configure.in
--- aMule-2.1.0.org/configure.in	2006-01-01 23:30:28.000000000 +0100
+++ aMule-2.1.0/configure.in	2006-01-02 22:02:24.636967250 +0100
@@ -10,8 +10,8 @@
 AM_MAINTAINER_MODE
 
 # Removing -fomit since it causes problems with CryptoPP
-CFLAGS="`echo $CFLAGS | sed "s/-fomit-frame-pointer//g"`"
-CXXFLAGS="`echo $CXXFLAGS | sed "s/-fomit-frame-pointer//g"`"
+#CFLAGS="`echo $CFLAGS | sed "s/-fomit-frame-pointer//g"`"
+#CXXFLAGS="`echo $CXXFLAGS | sed "s/-fomit-frame-pointer//g"`"
 
 # System detection, need AC_CANONICAL_TARGET
 case "${target_os}" in
