--- aMule-2.3.2/src/ClientCreditsList.cpp.orig	2016-09-16 09:55:06.000000000 +0200
+++ aMule-2.3.2/src/ClientCreditsList.cpp	2018-12-31 11:04:38.491753618 +0100
@@ -312,7 +312,7 @@
 		// calculate and store public key
 		CryptoPP::RSASSA_PKCS1v15_SHA_Verifier pubkey(*static_cast<CryptoPP::RSASSA_PKCS1v15_SHA_Signer *>(m_pSignkey));
 		CryptoPP::ArraySink asink(m_abyMyPublicKey, 80);
-		pubkey.DEREncode(asink);
+		pubkey.GetMaterial().Save(asink);
 		m_nMyPublicKeyLen = asink.TotalPutLength();
 		asink.MessageEnd();
 	} catch (const CryptoPP::Exception& e) {
