--- a/org.amule.aMule.desktop	2026-06-24 14:56:10.000000000 +0200
+++ b/org.amule.aMule.desktop	2026-08-23 12:31:59.532325756 +0200
@@ -4,7 +4,7 @@
 Icon=org.amule.aMule
 Terminal=false
 Type=Application
-Categories=Network;FileTransfer;
+Categories=GTK;Network;FileTransfer;
 StartupWMClass=org.amule.aMule
 GenericName=P2P File Sharing Client
 GenericName[es]=Cliente P2P de intercambio de archivos
@@ -15,6 +15,7 @@
 GenericName[tr]=P2P Dosya Paylaşım İstemcisi
 GenericName[zh_CN]=P2P 文件共享客户端
 Comment=A client for the eD2k network
+Comment[pl]=Klient sieci eD2k
 Comment[fr]=Un client pour le réseau eD2k
 Comment[tr]=eD2k ağı için istemci
 Comment[zh_CN]=eD2k网络客户端
