--- aMule-2.2.6/src/utils/wxCas/src/wxcasframe.cpp~	2008-09-06 17:39:00.000000000 +0200
+++ aMule-2.2.6/src/utils/wxCas/src/wxcasframe.cpp	2010-04-26 12:28:23.880637891 +0200
@@ -285,11 +285,11 @@
 #ifdef __WXMSW__
 
 	memdc.
-	SetFont ( wxFont::wxFont ( 6, wxSWISS, wxNORMAL, wxBOLD ) );
+	SetFont ( wxFont ( 6, wxSWISS, wxNORMAL, wxBOLD ) );
 #else
 
 	memdc.
-	SetFont ( wxFont::wxFont ( 8, wxSWISS, wxNORMAL, wxBOLD ) );
+	SetFont ( wxFont ( 8, wxSWISS, wxNORMAL, wxBOLD ) );
 #endif
 
 	memdc.
