diff -Nur old/po/Makefile.in.in new/po/Makefile.in.in
--- old/po/Makefile.in.in	2003-11-04 14:44:20.000000000 +0000
+++ new/po/Makefile.in.in	2003-11-15 17:36:20.000000000 +0000
@@ -27,7 +27,7 @@
 INSTALL = @INSTALL@
 INSTALL_DATA = @INSTALL_DATA@
 MKINSTALLDIRS = @MKINSTALLDIRS@
-mkinstalldirs = $(SHELL) `case "$(MKINSTALLDIRS)" in /*) echo "$(MKINSTALLDIRS)" ;; *) echo "$(top_builddir)/$(MKINSTALLDIRS)" ;; esac`
+mkinstalldirs = $(SHELL) $(MKINSTALLDIRS)
 
 GMSGFMT = @GMSGFMT@
 MSGFMT = @MSGFMT@
